core/src/main/resources/jenkins/slaves/systemInfo/ClassLoaderStatisticsSlaveInfo/systemInfo.groovy

```diff
@@ -1,20 +1,21 @@
 import hudson.slaves.SlaveComputer
 
 def fmt = new java.text.DecimalFormat("0.0")
-def right = 'text-align: right'
 if (my instanceof SlaveComputer) {
     SlaveComputer c = my
 
-    table(class: 'bigtable') {
-        tr {
-            th _('Loading Type')
-            th _('Time (s)')
-            th _('Count')
+    table(class: 'jenkins-table') {
+        thead {
+            tr {
+                th { text(_('Loading Type')) }
+                th { text(_('Time (s)')) }
+                th { text(_('Count')) }
```

Comment:
Why this change?

Reply:
specifically? count is there before

Reply:
The change from `th _('key')` to `th { text(_('key')) }`

Reply:
I made that change because I didn't find a way to make it render correctly with one of the iterations that was using

```
th(style: right) _('Time (s)')
```

When that wouldn't work for me (non-existent Javascript / CSS skills), I switched to something that worked for me:

```
th(style: right) { text(_('Time (s)')) }
```

That was based on entries elsewhere in the file.

I'm more than happy to restore it to the earlier form, especially if there is a benefit from that change:

```
th _('Time (s)')
```

Is there a benefit from returning it to the original text?